habitat-baselines/habitat_baselines/rl/hrl/hierarchical_policy.py

```diff
@@ -190,7 +203,9 @@ def num_recurrent_layers(self):
         if self._high_level_policy.num_recurrent_layers != 0:
             return self._high_level_policy.num_recurrent_layers
         else:
-            return self._skills[0].num_recurrent_layers
+            return self._skills[
```

Comment:
When is 0 not in _skills? Or it is for generality?

Reply:
`0` would actually still work here. I originally tried changing the index to based on PyTorch tensors, but even without that change, I think this is still best for generality (since we just want to get any skill here).